packages/open-truss/tsconfig.json

```
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

https://www.typescriptlang.org/tsconfig#target

```suggestion
    "target": "es6",
```